### PR TITLE
hydronic system additions / edits

### PR DESCRIPTION
**Pull request by evgeniya-engel**

### CODE REVIEW

**Comment by evgeniya-engel** [comment]
@shambergoldstein @ccquigley ready for your review! I verified that the edited types are only used in only these assets that they were created fore and no other assets will be affected.

File: ontology/yaml/resources/units/units.yaml [hunk 1]
```diff
@@ -102,6 +102,8 @@ distance:
   inches:
     multiplier: 0.0254
     offset: 0
+thermalefficiency:
+  kilowatts_per_ton: STANDARD
```
Comment (shambergoldstein) [inline ontology/yaml/resources/units/units.yaml:106]: The telemetry field looks to be on a scale of 0 to 100 and given that it is an efficiency I would suspect that the units should be percent - please confirm

**Review by shambergoldstein — CHANGES_REQUESTED** [review]
Note: backward incompatible changes in this PR only effect the specific devices that this PR is for.

File: ontology/yaml/resources/units/units.yaml (hunk 1 above)
Comment (evgeniya-engel) [reply ontology/yaml/resources/units/units.yaml:106]: It should be kW/ton. I adjusted value range to all flexible between 0 and 10 kw/ton.

**Review by shambergoldstein — APPROVED** [review]

**Pull request merged**